resources/extruders/KOONOVO_ELF.def.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "KOONOVO_ELF",
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "KOONOVO",
```

Comment:
This refers to a .def.json that does not exist. I think you mean KOONOVOELF

Reply:
"machine": "KOONOVOELF",